lib/article/article.types.ts

```diff
@@ -0,0 +1,92 @@
+import { Timestamp } from '../common/common.types';
+
+export type ArticleObject = {
+    type: 'article';
+    id: string;
+    workspace_id: string;
+    title: string;
+    description: string;
+    body: string;
+    author_id: string;
+    state: string;
+    created_at: Timestamp;
+    updated_at: Timestamp;
+    url: string;
+    parent_id: string;
+    parent_type: string;
+    default_locale: string;
+    translated_content: TranslatedContentObject;
+    statistics: StatisticsObject;
+};
+
+export enum Locales {
```

Comment:
doesn't typescript have a type for this? seems like something that should be there

Reply:
AFAIK, no :)